startup/20-area-detectors.py

```diff
@@ -648,8 +648,8 @@ def enable_plugins(det, plugin_names):
     set_eiger_defaults(eiger500k_single)
     # AD v3.3+ config:
     eiger500k_single.cam.ensure_nonblocking()
-    eiger500k_single.file.write_path_template = '/nsls2/data/chx/assets/eiger500k/%Y/%m/%d/'
-    eiger500k_single.file.reg_root ='/nsls2/data/chx/assets/eiger500k/'
+    eiger500k_single.file.write_path_template = f'{proposal_dir}/assets/eiger500k/%Y/%m/%d/'
+    eiger500k_single.file.reg_root =f'{proposal_dir}/assets/eiger500k/'
```

Comment:
Have you considered making it `assets_dir`? That would make it harder to accidentally write into the top-level proposals dir (where users have access to move and delete things).